src/views/labs/LabView/LabView.vue

```diff
@@ -61,20 +58,20 @@
       return this.pageData?.lab;
     }
 
-    roundClosed(round) {
+    roundClosed(round:{round:number}) {
       return (
-        round.round < this.lab.puzzles.length
-        || this.lab.exp_phase == null
-        || this.lab.exp_phase >= 1
+        round.round < Number(this.lab?.puzzles?.length)
+        || this.lab?.exp_phase == null
+        || Number(this.lab?.exp_phase) >= 1
       );
     }
 
     get closedRounds() {
-      return this.lab.puzzles.filter(round => this.roundClosed(round));
+      return this.lab?.puzzles.filter(round => this.roundClosed(round));
     }
 
     get openRounds() {
-      return this.lab.puzzles.filter(round => !this.roundClosed(round));
+      return this.lab?.puzzles.filter(round => !this.roundClosed(round));
```

Comment:
This seems to create a lot of new errors in the template; Adding `??` at the end fixes them.
(Same for the previous function)